.github/workflows/integration-test.yaml

```diff
@@ -143,6 +143,9 @@ jobs:
           ref: ${{ steps.vars.outputs.sha }}
           persist-credentials: false
 
+      - name: Cleanup Disk space in runner
+        uses: ./.github/actions/disk-cleanup
+
```

Comment:
Shouldn't we run this from TRUSTED code? Meaning, before line 138 cc @nbusseneau

Reply:
I was on the fence, but just feel like it's not required for cilium members (assume that upstream workflow will be used for external contributor).